pkg/api/v1/config.go

```diff
@@ -122,3 +123,75 @@ func (s Server) GetDashboardGroup(w http.ResponseWriter, r *http.Request) {
 	}
 	http.Error(w, fmt.Sprintf("Dashboard group %q not found", vars["dashboard-group"]), http.StatusNotFound)
 }
+
+// ListDashboards returns every dashboard in TestGrid
+// Response Proto: ListDashboardResponse
```

Comment:
It looks like this writes json objects, not protos?

Reply:
I think we just refer to the response type in the proto file.

Reply:
But it isn't a response proto, it is a response JSON

Reply:
I see. I will create another PR to resolve this in all config APIs